### PR TITLE
initial added restrictedpython recipe

### PR DESCRIPTION
**Pull request by BubblyOrca**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/restrictedpython/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  home: https://github.com/zopefoundation/RestrictedPython
+  summary: RestrictedPython is a defined subset of the Python language which allows to provide a program input into a trusted environment.
+  dev_url: https://github.com/zopefoundation/RestrictedPython
+  license: ZPL-2.1
+  license_file: COPYRIGHT.txt
+
+extra:
+  recipe-maintainers:
+    - BubblyOrca
+    - mxr-conda
```
Comment (synapticarbors) [inline recipes/restrictedpython/meta.yaml:43]: Could you confirm that this co-maintainer agrees to be listed?

**Review by synapticarbors — COMMENTED** [review]
Looks good to me other than resolving my one comment.

File: recipes/restrictedpython/meta.yaml (hunk 1 above)
Comment (mxr-conda) [reply recipes/restrictedpython/meta.yaml:43]: Agree to maintain

File: recipes/restrictedpython/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/zopefoundation/RestrictedPython
+  summary: RestrictedPython is a defined subset of the Python language which allows to provide a program input into a trusted environment.
+  dev_url: https://github.com/zopefoundation/RestrictedPython
+  license: ZPL-2.1
+  license_file: COPYRIGHT.txt
```
Comment (dopplershift) [inline recipes/restrictedpython/meta.yaml:38]: Need this and the license file:
```suggestion
  license_file:
    - COPYRIGHT.txt
    - LICENSE.txt
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Fix up the license files and this is good to go.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**